apps/fxc-front/src/app/logic/track.ts

```diff
@@ -1,9 +1,21 @@
-import { extractGroupId } from '@flyxc/common';
+import {extractGroupId, RuntimeTrack} from '@flyxc/common';
 
 import { unwrapResult } from '@reduxjs/toolkit';
 
-import { store } from '../redux/store';
-import { fetchTrack } from '../redux/track-slice';
+import {AppDispatch, store} from '../redux/store';
+import { currentLeague } from '../redux/selectors';
+import {fetchTrack, RuntimeTrackId, setScore as setTrackStore} from '../redux/track-slice';
+// @ts-ignore
+import ScoreWorker from '../workers/score-track?worker';
+import {Request as ScoreRequest, Response as ScoreResponse} from '../workers/score-track'
+import {
```

Comment:
If you find yourself having to do too much renaming you can use a named import, as in

```
import * as plannerSlice from '../../redux/planner-slice';
```

(I think VSCode has an automated refactoring for that)